app/controllers/public/index.js

```diff
@@ -4,11 +4,15 @@ const { Controller, computed, String } = Ember;
 
 export default Controller.extend({
 
-  speakers: [{  name: 'Speaker 1', organisation: 'Org 1' },
-  { name: 'Speaker 2', organisation: 'Org 2', socialLinks: [{ name: 'facebook', url: '#' }] },
-  { name: 'Speaker 3', organisation: 'Org 3', socialLinks: [{ name: 'linkedin', url: '#' }] },
-  { name: 'Speaker 4', organisation: 'Org 4', socialLinks: [{ name: 'twitter', url: '#' }] },
-  { name: 'Speaker 5', organisation: 'Org 5', socialLinks: [{ name: 'home', url: '#' }] }],
+  speakers: [{  name: 'Speaker 1', organisation: 'Org 1', isFeatured: false, socialLinks: [{ name: 'facebook', url: '#' }]  },
+  { name: 'Speaker 2', organisation: 'Org 2', isFeatured: false, socailLinks: [{ name: 'facebook', url: '#' }] },
+  { name: 'Speaker 3', organisation: 'Org 3', isFeatured: true, socialLinks: [{ name: 'linkedin', url: '#' }], short_biography: 'Full-stack developer with experience in PHP, Python &amp; Ember.JS. Plays around with Kubernetes and Containers. Avid open source contributor.' },
+  { name: 'Speaker 4', organisation: 'Org 4', isFeatured: true, socialLinks: [{ name: 'twitter', url: '#' }], short_biography: 'Full-stack developer with experience in PHP, Python &amp; Ember.JS. Plays around with Kubernetes and Containers. Avid open source contributor.' },
+  { name: 'Speaker 5', organisation: 'Org 5', isFeatured: false, socialLinks: [{ name: 'home', url: '#' }],  short_biography: 'Full-stack developer with experience in PHP, Python &amp; Ember.JS. Plays around with Kubernetes and Containers. Avid open source contributor.' }],
+
+  filterByFeaturedSpeakers: computed.filterBy('speakers', 'isFeatured', true),
```

Comment:
Let the variable names be simple and as close to english language as possible.
`featuredSpeakers` and `nonFeaturedSpeakers`